src/index.ts

```diff
@@ -151,15 +160,19 @@ export default class FileOrganizer extends Plugin {
     content: string,
     name: string
   ): Promise<{ type: string; formattingInstruction: string } | null> {
+    if (!this.settings.aiTemplateFormatting) {
+      logMessage("AI Template Formatting is disabled");
+      return null;
+    }
     const classifications = await this.getClassifications();
     logMessage("classifications", classifications);
 
     const prompt = `Name: ${name}
-  Content:
-  ${content}
-  classifications:${classifications.map((c) => c.type).join(", ")}
-Which of the following classifications would 
-  be the most appropriate for the given content?`;
+          Content:
+          ${content}
```

Comment:
make sure you configured your vscode to use prettier (command pallete - format document with - configure default formatter)

Reply:
addressed